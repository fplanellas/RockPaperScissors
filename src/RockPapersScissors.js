import GameScreen from "./components/GameScreen";

const RockPapersScissors = () => {
  return (
    <GameScreen />
  )
}

export default RockPapersScissors
