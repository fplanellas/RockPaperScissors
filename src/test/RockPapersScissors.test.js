import React from 'react';
import { shallow } from 'enzyme';
import RockPapersScissors from '../RockPapersScissors';

describe('testing in main component RockPapersScissors', () => {

    test('should be displayed correctly', () => {
        const wrapper = shallow( <RockPapersScissors /> );
        expect( wrapper ).toMatchSnapshot();
    })
    
})
