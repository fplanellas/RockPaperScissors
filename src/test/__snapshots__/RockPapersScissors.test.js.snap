// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing in main component RockPapersScissors should be displayed correctly 1`] = `<GameScreen />`;
